### DISCUSSION TITLE
Unable to launch a new process

### DISCUSSION THREAD

**Discussion by fafayqa** [composer/composer#12008] [Q&A / Support]
I am trying to install [endroid qr-code](https://github.com/endroid/qr-code) php library using Composer FOR THE FIRST TIME EVER but whatever I try it always give an error: `Unable to launch a new process`.

I am using kind of "manual install trick" shown [HERE](https://stackoverflow.com/a/48319499/19018243) as I do not have option running custom install scripts on a live server, I can only manually upload files.

So I did exactly step-by-step what was required, yet I did not move a bit as it immediately throws at me abovementioned error in a big red square in CMD window as soon as I run CMD command `php composer.phar update` (as required).

I am on Win10 x64 using USBWebserver with PHP 8.1 and have all the file access privilegues.

**Comment by Seldaek** [comment]
That looks like proc_open is not allowed on your server.. Which is not something Composer related really in any case as application-runtime errors are not related to Composer but the exact packages you are using.

**Reply by fafayqa** [reply]
Ehm, sorry, I am no pro, so can you say it in some more common way so I would have at least some hint as where the problem is and what to do? Maybe how to enable proc_open on my server (USBWebserver8.1)?

**Reply by Seldaek** [reply]
The point is from what it *seems* to me (because I got very little details to work from..) that your web host disabled some php functions like proc_open, and now you are screwed. Either you need to enable that and perhaps they offer an option to do so, or you'd need to use another web host, or another library for QR codes that does not need new processes.

**Reply by fafayqa** [reply]
What details you need to know to help me more? :-) Once again: I am on a local machine, PC, local testserver using USBWebserver8.1 as server where I develop webpages. Is there some settings in php.ini of the server which needs to be set/enabled/disabled or what?

**Reply by fafayqa** [reply]
BTW I just search in php.ini and on line `disable_functions = ` there is nothing after `=`, so maybe you are probably wrong? Also: why did you set this as CLOSED? Where was this solved/successfully answered? Or did I miss something? Confused...

**Reply by Seldaek** [reply]
I closed because it has nothing to do with Composer. I'm trying to help here but I am not familiar with USBWebserver, nor very familiar with endroid/qr-code.. So yeah hopefully someone can help you on the issue you created over there sorry.
